<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingClass</class>
 <widget class="QWidget" name="settingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>672</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="setting.qrc">
    <normaloff>:/setting/Resources/xidian1.png</normaloff>:/setting/Resources/xidian1.png</iconset>
  </property>
  <widget class="QLineEdit" name="lineEdit_ExposureTime">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>96</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_AcquisitionMode">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>34</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>Single</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kinetic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Video</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>338</x>
     <y>64</y>
     <width>271</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Binning ：</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_Binning1">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>150</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>150</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Binning2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>128</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_Binning1">
       <property name="text">
        <string>1*1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_Binning2">
       <property name="text">
        <string>2*2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_Binning4">
       <property name="text">
        <string>4*4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_Binning8">
       <property name="text">
        <string>8*8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_Binning16">
       <property name="text">
        <string>16*16</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_BinningCustom">
       <property name="text">
        <string>Custom</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>153</y>
     <width>114</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Shutter Opens On ：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_TriggerMode">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>210</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_Shutter">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>153</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ON</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OFF</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>96</y>
     <width>132</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Exposure Time[secs] ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>4</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Acquisition Mode ：</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>560</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="pushButton_SetAcqParameter">
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CancelAcqParameter">
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>103</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Triggering Mode ：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>271</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Electron Multiplier (EM) Gain</string>
   </property>
   <widget class="QCheckBox" name="checkBox_EMEnable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_EMAdvanced">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>23</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Advanced</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Electron Multiplier Gain Level</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_EMRange">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>311</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Kinetic ：</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Kinetic2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Kinetic Mode 2 ：</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>291</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>23</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Kinetic Cycle Time ：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_KineticCycleTime_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>23</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>80</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>The number Kinetic ：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_KineticNumber">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_KineticTime">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_KineticCycleTime">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>23</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>80</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Kinetic Time ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>23</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Kinetic Cycle Time ：</string>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="radioButton_Kinetic1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kinetic Mode 1 ：</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="setting.qrc"/>
 </resources>
 <connections/>
</ui>
